src/kitovu/sync/settings.py

```diff
@@ -110,25 +110,22 @@ class Settings:
     }
 
     @classmethod
-    def from_yaml_file(cls, path: typing.Optional[pathlib.Path] = None,
-                       validator: typing.Optional[utils.SchemaValidator] = None) -> 'Settings':
```

Comment:
I removed the ability to pass a custom validator because we didn't do that anywhere anyways (not even in tests).